src/vcpkg/install.cpp

```diff
@@ -1237,21 +1243,40 @@ namespace vcpkg::Install
         Install::perform_and_exit(args, paths, default_triplet, host_triplet);
     }
 
-    SpecSummary::SpecSummary(const PackageSpec& spec, const Dependencies::InstallPlanAction* action)
-        : spec(spec), build_result{BuildResult::NULLVALUE, nullptr}, action(action)
+    SpecSummary::SpecSummary(const Dependencies::InstallPlanAction* action)
```

Comment:
Given that `action` must not be `nullptr`, why isn't it passed with a reference (now) instead of a pointer?

Reply:
Done.